package.json

```diff
@@ -24,5 +24,11 @@
           "name": "Stan's Assets",
           "email": "stan@stansassets.com",
           "url": "https://stansassets.com/"
+    },
+    "dependencies": {
+      "com.unity.test-framework": "0.0.4-preview",
+      "com.unity.modules.imgui": "1.0.0",
+      "com.unity.modules.jsonserialize": "1.0.0",
+      "com.unity.modules.imageconversion": "1.0.0"
```

Comment:
looks good, but the fact that `test-framework` preview version is requested a bit weird.
I am wondering what would happen if we will install this package to 2019 or 2020 Unity version.

Will it grab the latest available & verified `test-framework` or it will fail to resolve? or it will grab the `0.0.4-preview`

Reply:
Also, why do we need `com.unity.modules.imgui` we should not give this dependency in the foundation